src/wsiRequest.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SRC_WSIREQUEST_H_
+#define SRC_WSIREQUEST_H_
+
+#include <string>
+#include "src/enums.h"
+
+// Structure for wsi2dcm settings
+// inputFile - wsi input
```

Comment:
Put these comments on the actual member variables within the struct

Reply:
done